cves/CVE-2007-1862.yml

```diff
@@ -111,8 +115,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
-  name:
+  answer: cache
```

Comment:
For "answer" put how you determined this.

Reply:
I tried fixing this in a commit a few days ago, forgot to respond to the comment.